### ISSUE TITLE
Design Feedback and Contribution

### ISSUE THREAD

**Issue by jmgilman** [jmgilman/ndc#2]
The Nix Dev Container project is currently seeking core contributors who are interested in maintaining this project going forward. Anyone is welcome to join as a core contributor, however, the following skillsets will make contributing easier:

1. Familiarity with [Docker](https://www.docker.com/), specifically [devcontainers](https://code.visualstudio.com/docs/remote/containers).
2. Familiarity with [Nix](https://nixos.org/), especially managing Nix-based projects. 
3. Experience working with development environments.

Note that this is an open-source project, and therefore contributions are welcome from anyone. Core contributors differ in that they will directly speak into the design of NDC and will be given write access to the repository. 

If you're interested in becoming a core contributor, [please join us on Gitter](https://gitter.im/nix-dev-container/community?utm_source=share-link&utm_medium=link&utm_campaign=share-link). 

The design document is [currently being finalized](https://github.com/jmgilman/ndc/pull/1) and feedback is appreciated. You can see a rendered version of it [here](https://github.com/jmgilman/ndc/blob/fbbd07e70452378d52c19cdb1edc0f13cc620ecb/DESIGN.md).

**Comment by aszenz** [comment]
I'm interested in these set of technologies, but what is the exact project, is it still to be built?

**Comment by jmgilman** [comment]
Yes, the project is still in the design phase. The design document is a WIP and covers the details of the goals for the project. Once it's finished then work will begin on the project itself.
